src/supermarket/spec/helpers/application_helper_spec.rb

```diff
@@ -46,4 +46,17 @@
       expect(flash_message_class_for("warning")).to eql("warning")
     end
   end
+
+  describe "#github_profile_url" do
+    ENV["GITHUB_ENTERPRISE_URL"] = "https://example.com"
```

Comment:
can we keep this in secrets file of test env?

Reply:
Yes I also tried for the same but it's not taking value from test env .

Reply:
ok